lib/core-classes/DeviceService.ts

```diff
@@ -250,6 +250,11 @@ export class DeviceService {
     asset._source.measures = measures;
     device._source.assetId = linkRequest.assetId;
 
+    if(!Array.isArray(asset._source.deviceIds)) {
+      asset._source.deviceIds=[];
+    }
```

Comment:
Coding standards. Can you modify the linter config to match our standard on this projet? You need to copy those files
  - https://github.com/kuzzleio/kuzzle/blob/2-dev/.eslintrc.json
  - https://github.com/kuzzleio/kuzzle/blob/2-dev/.eslintc-ts.json
 
 ```suggestion
    if(! Array.isArray(asset._source.deviceIds)) {
      asset._source.deviceIds = [];
    }
```

Reply:
done